tests/app/test_users.py

```diff
@@ -181,6 +190,20 @@ def test_return_400_for_invalid_user_json(self):
                 'users': {
                     'email_address': 'joeblogs@email.com',
                     'password': '',
+                    'role': 'buyer',
+                    'name': 'joe bloggs'}}),
+            content_type='application/json')
+
+        assert_equal(response.status_code, 400)
```

Comment:
I think it's worth checking for a specific error message as well as the response code - there are a lot of ways to get a 400 out of the system, and we should be confident that the request has failed for the right reason.